views/partials/header.ejs

```diff
@@ -1,25 +1,25 @@
-<h1>The Jobs EJS Application</h1>
+<!-- <div style="display: flex; flex-direction: column; align-items: center; justify-content: center; height: auto;"> -->
+<h1 style="text-align: center;">The Jobs EJS Application</h1>
 <% if (user) { %>
-   <p>User <%= user.name %> is logged on.</p>
-   <form method="POST" action="/sessions/logoff">
-   <button>Logoff</button>
+  <p style="color: green; text-align: center; margin-top: 5%;">User <%= user.name %> is logged on.</p>
+   <form method="POST" action="/sessions/logoff" style="position: absolute; top: 0; right: 0;">
+   <button style="background-color: darkgreen; color: white;">Log out</button>
    </form>
 <% } %>
-<% if (errors) {
+<% if (errors && errors.length) {
     errors.forEach((err) => { %>
-      <div>
-        Error: <%= err %>
-      </div>
+      <div class="alert alert-danger" style="width: 30%; margin: 0 auto;">Error: <%= err %></div>
```

Comment:
Nitpick:
Elements that contain text should leverage semantic HTML. For example, this alert should be a `p` tag because its role is to display text.